docs/pages/kubernetes-access/controls.mdx

```diff
@@ -1,236 +1,120 @@
-    attribute statements and claims received from identity providers to Teleport's roles.
-
-    ```yaml
-    kind: saml
-    version: v2
-    metadata:
-      name: okta
-    spec:
-      acs: https://tele.example.com/v1/webapi/saml/acs
-      attributes_to_roles:
-      # Any user who has SAML `groups` attribute with value `okta-admin`
-      # will be given the Teleport `kube-access` role.
-      - {name: "groups", value: "okta-admin", roles: ["kube-access"]}
-      entity_descriptor: |
-        <?xml !!! Make sure to shift all lines in XML descriptor
-        with 4 spaces, otherwise things will not work
-    ```
-
-    In this example, the `kube-access` role uses the `kubernetes_groups` property to assign Teleport users to [Kubernetes groups](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#referring-to-subjects)
-
-    ```yaml
-    kind: role
-    version: v3
-    metadata:
-      name: kube-access
-    spec:
-      allow:
-        # If kubernetes integration is enabled, this setting configures which
-        # kubernetes groups the users of this role will be assigned to.
-        # Note that you can refer to a SAML/OIDC trait named 'property' via the "{{external.property}}" template variable,
-        # this allows you to specify Kubernetes group membership via your identity provider:
-        kubernetes_groups: ["system:masters"]
-    ```
-  </TabItem>
-</Tabs>
-
-## Mapping OIDC claims and SAML attributes to Kubernetes groups
-
-Identity provider admins can assign metadata to a user, such as
-group membership or access permissions. Administrators configure what metadata
-is shared with Teleport. Teleport receives user metadata keys and values as OIDC claims or SAML
-attributes during [single sign-on redirect flow](https://goteleport.com/blog/how-oidc-authentication-works/):
-
-```yaml
-# Alice has an email alice@example.com. Email is a standard OIDC claim.
-email: "alice@example.com"
-# Alice is a member of groups admins and devs
-groups: ["admins", "devs"]
-# She can access prod and staging environments
-access: {"env": ["prod", "staging"]}
-```
-
-Teleport's roles map OIDC claims or SAML attributes using template variables.
-Template variable `{{external.groups}}` will
-be substituted to the users' group list in the role definition.
+You can add labels to a Kubernetes cluster when you register it with Teleport.
+You can restrict a user's access to Kubernetes clusters with different labels
+using a role's  `kubernetes_labels` field.
 
 ```yaml
 kind: role
-version: v5
 metadata:
-  name: group-member
+  name: kube-access
+version: v6
 spec:
   allow:
-    # For Alice, will be substituted with the list ["admins", "devs"]
-    kubernetes_groups: ["{{external.groups}}"]
-    # For Alice, will be substituted with ["alice@example.com"]
-    kubernetes_users: ["{{external.email}}"]
+    kubernetes_labels:
+      'region': '*'
+      'environment': 'development'
```

Comment:
maybe it's worth mentioning that a label key supports multiple values
`'key': ['val1','val2']`
You are allowed to access that cluster if the Kubernetes cluster has a label with `key=key` whose value is `val1` or `val2`